sdks/java/container/license_scripts/dep_urls_java.yaml

```diff
@@ -48,5 +48,5 @@ paranamer:
   '2.7':
     license: "https://raw.githubusercontent.com/paul-hammant/paranamer/master/LICENSE.txt"
 xz:
-  '1.8':
-    license: "https://git.tukaani.org/?p=xz-java.git;a=blob_plain;f=COPYING;h=c1d404dc7a6f06a0437bf1055fedaa4a4c89d728;hb=HEAD"
+  '1.5':
+    license: "https://git.tukaani.org/?p=xz-java.git;a=blob;f=COPYING;h=c1d404dc7a6f06a0437bf1055fedaa4a4c89d728;hb=9f1f97a26f090ffec6568c004a38c6534aa82b94"
```

Comment:
For sdks:java:container:pullLicenses to task to succeed, i had to downgrade xz licence version from 1.8 to 1.5 as suggested by the build scan https://scans.gradle.com/s/2tv254pmdrzxq/console-log?task=:sdks:java:container:pullLicenses